src/helpers/helpers.options.ts

```diff
@@ -203,6 +203,8 @@ export function _addGrace(minmax: { min: number; max: number; }, grace: number |
  * @param context
  * @returns
  */
-export function createContext<P extends T, T extends object>(parentContext: P, context: T): P extends null ? T : P & T {
+export function createContext<T extends object>(parentContext: null, context: T): T;
+export function createContext<T extends object, P extends T>(parentContext: P, context: T): P & T;
+export function createContext(parentContext: object, context: object) {
```

Comment:
isn't this a bit ugle and basicly the same as any since everything in js is an object?

Reply:
I'm used this because generic type extends `object`

Reply:
@LeeLenaleee `any` is denied by eslint, I can replace it with `Record<string, unknown>`, but it is an internal realization of the above overloadings, so it doesn't matter. Overloadings are required because in the previous realization

```
createContext<P extends T, T extends object>(parentContext: P, context: T)
```

if `parentContext = null` then `P = any`, and `any & T = any`.